sagemaker_processing/feature_transformation_with_sagemaker_processing_Dask/container/program/bootstrap.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import os
+import socket
+import json
+import psutil
+import subprocess
+import sys
+import time
+from dask.distributed import Scheduler, Worker, Client
+from shutil import copyfile
+from subprocess import Popen, PIPE
+
+
```

Comment:
nit: general whitespace issues here and elsewhere. should probably apply an auto-formatter

Reply:
@metrizable I just cleaned the code format using black and pushed the code.